packages/cosmo/package.py

```diff
@@ -61,7 +61,8 @@ class Cosmo(MakefilePackage):
     patch('patches/5.07.mch1.0.p4/patch.Makefile', when='@5.07.mch1.0.p5')
 
     dycore_deps(apngit)
-    dycore_deps(c2smgit)
+    # Uncomment to build c2sm release 
+    # dycore_deps(c2smgit)
```

Comment:
I agree under the condition that we find a suitable solution for this problem soon.

Reply:
thanks @jonasjucker . I will merge it, provided we need to find a proper solution soon